### PR TITLE
Feat: Make Excon instance to accept `ciphers` SSL option (#45)

### PR DESCRIPTION
**Pull request by saiqulhaq**
This PR is to support ciphers SSL Option usage as mentioned in this PR https://github.com/lostisland/faraday/issues/1586

### CODE REVIEW

File: lib/faraday/excon/version.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 module Faraday
   module Excon
-    VERSION = '2.1.0'
+    VERSION = '2.2.0'
```
Comment (olleolleolle) [inline lib/faraday/excon/version.rb:5]: I think perhaps the release process does the version upgrade of this library, best keep that out of the PR.

**Comment by saiqulhaq** [comment]
Thanks for the feedback @olleolleolle 

I just updated the Faraday dependency to be more permissive. 
FYI following is the RSpec result
image

**Comment by olleolleolle** [comment]
@saiqulhaq Great! Now, let's see what the maintainers of this repo say. I recommend a merge!

File: lib/faraday/excon/version.rb (hunk 1 above)
Comment (geemus) [reply lib/faraday/excon/version.rb]: Yep, thanks for letting them know about that.

**Comment by geemus** [comment]
LGTM, thanks!

**Comment by geemus** [comment]
Does this need a release?

**Comment by olleolleolle** [comment]
@geemus Well, I don’t think anybody’s waiting for precisely this, but it’s a nice feature. Take it in your time.

**Pull request merged**